standup/status/views.py

```diff
@@ -124,6 +125,22 @@ def statusize(request):
     return HttpResponseRedirect(data.get('redirect_to', referrer))
 
 
+class SearchView(PaginateStatusesMixin, TemplateView):
+    template_name = 'status/search.html'
+
+    def get_status_queryset(self):
+        query = self.request.GET.get('query')
+        if not query:
+            return []
+
+        return Status.objects.filter(generate_query('content', query))
```

Comment:
Because of the way this was implemented, we can trivially wrap an API around it. Yay!